src/styles/globals.css

```diff
@@ -2,6 +2,12 @@
 @tailwind components;
 @tailwind utilities;
 
+@layer utilities {
+	.text-shadow-none {
+		text-shadow: none;
+	}
+}
+
```

Comment:
is this applied on all stories? Or what does this do exactly?

Reply:
It adds an extra tailwind utility class. So you can use it like this
```js
<div className="text-shadow-none"></div>
```

You can read more about it here: https://tailwindcss.com/docs/functions-and-directives#layer